bool[] status
bool[] is_safe
bool[] is_valid
int32 active_monitoring_case
